YoCode/Directory.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace YoCode
+{
+
+    
+    public class Directory
+    {
+        public List<String> OriginalPaths { get; } = null;
+        public List<String> ModifiedPaths { get; } = null;
```

Comment:
You don't need to initialise these to `null`, that's the default for a reference type. And you set them in the constructor anyway.